bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/ephemeris/EphemerisFiles.scala

```diff
@@ -0,0 +1,125 @@
+package edu.gemini.dbTools.ephemeris
+
+import edu.gemini.dbTools.ephemeris.ExportError.FileError
+import edu.gemini.spModel.core.HorizonsDesignation
+
+import java.net.{URLDecoder, URLEncoder}
+import java.nio.charset.StandardCharsets.UTF_8
+import java.nio.file.{Path, Files}
+import java.time.Instant
+
+import scala.collection.JavaConverters._
+
+import scalaz._, Scalaz._
+
+/** Support for working with ephemeris files in a directory. */
+sealed trait EphemerisFiles {
```

Comment:
Hopefully the TCS will read the files on a mounted file system instead of looking in its installation directory as it does today.